app/src/main/java/org/mozilla/focus/utils/Settings.kt

```diff
@@ -176,7 +176,7 @@ class Settings(
                 .commit()
         }
 
-    var isCfrForForShieldToolbarIconVisible: Boolean
+    var shouldShowCfrForShieldToolbarIcon: Boolean
```

Comment:
👍
This seems to express more clearly what the value represents and where it can be used.